lib/sentimental.rb

```diff
@@ -38,24 +38,34 @@ def classify(string)
   end
 
   def load_defaults
-    ['sentiwords', 'sentislang'].each do |filename|
-      load_from(File.dirname(__FILE__) + "/../data/#{filename}.txt")
+    %w(slang en_words).each do |filename|
+      load_from_json(File.dirname(__FILE__) + "/../data/#{filename}.json")
     end
   end
 
   def load_from(filename)
+    word_scores.merge!(self.class.hash_from_file(filename))
+  end
+
+  def self.hash_from_file(filename)
```

Comment:
Yes I needed that method outside the class, extracting it to a new module would be great because I don't think it's related to this class. I will work on it